### PR TITLE
rosAFE section should now be OK

### PR DESCRIPTION
**Pull request by sylvarg**
All the sections dedicated to rosAFE should now be there. Hope everything is OK.

### CODE REVIEW

File: robo/rosafe/rosafe.txt [hunk 1]
```diff
@@ -198,50 +327,460 @@ Requesting processors
...
+  .. table:: Input parameters of the ``Gammatone`` service of RosAFE
+
+    +---------------+------------------+---------------+----------------------------------------------+
+    | Name          | Data type        | Default value | Documentation                                |
+    +===============+==================+===============+==============================================+
+    | name          | string           | gammatone     | The name of this activity                    |
+    +---------------+------------------+---------------+----------------------------------------------+
+    | upperDepName  | string           | preProc       | The name of the upper dependency             |
+    +---------------+------------------+---------------+----------------------------------------------+
+    | **fb_type**   | string           | string        | Filterbank type (gammatone or drnl)          |
```
Comment (hagenw) [inline robo/rosafe/rosafe.txt]: Is this written in bold font to highlight that it is the most important factor?
Or is it a mistake and should be written in normal text?

File: robo/rosafe/matlab_client.txt [hunk 2]
```diff
@@ -3,4 +3,211 @@
...
+
+The following script will continuously plot the results of the ILD processor coming from the RosAFE.
+
+.. code-block:: Matlab
+
+  clear all; close all; clc;
+
+  openRobotsMatlabPath = '~/openrobots/lib/matlab';
+  twoEarsPath = '~/TwoEars/AuditoryModel/TwoEars-1.2/';
+  rosAFE_matlab_Path = '~/genom_ws/rosAFE/matlabFiles';
```
Comment (hagenw) [inline robo/rosafe/matlab_client.txt:165]: Maybe we should add a comment before these paths that the user has to adjust them?

**Review by hagenw — COMMENTED** [review]
Thanks, it compiles without any errors and looks fine.
I haven't checked for spell errors. If you still want to do this, please tell me. Otherwise I would going to merge it, after you answered my two small comments.

File: robo/rosafe/rosafe.txt (hunk 1 above)
Comment (sylvarg) [reply robo/rosafe/rosafe.txt]: It's a mistake, you can remove the bold font here.

File: robo/rosafe/matlab_client.txt (hunk 2 above)
Comment (sylvarg) [reply robo/rosafe/matlab_client.txt:165]: You're right, maybe we can add something like : "Depending on your configuration and installation, you might have to adapt the following to point to the correct paths"
Do you add it yourself or do you want me to push it in the rosafe branch?

Comment (hagenw) [reply robo/rosafe/matlab_client.txt:165]: Would be nice if you could push both small corrections, thanks.

Comment (sylvarg) [reply robo/rosafe/matlab_client.txt:165]: Done.

**Pull request merged**